src/flyout_cursor.ts

```diff
@@ -48,8 +63,17 @@ export class FlyoutCursor extends Blockly.Cursor {
    *
    * @returns Always null.
    */
-  override in(): null {
-    return null;
+  override in(): Blockly.ASTNode | null {
+    const curNode = this.getCurNode();
```

Comment:
I think I'm missing why in/out have been added here. I think they will never be called by `NavigationController` in `FLYOUT` state. They do no harm so far as I can tell.